plugin/trino-delta-lake/README.md

```diff
@@ -0,0 +1,191 @@
+#Delta Lake Connector Developer Notes
+
+The Delta Lake connector can be used to interact with [Delta Lake](https://delta.io/) tables.
+
+Trino has product tests in place for testing its compatibility with the 
+following Delta Lake implementations:
+
+- Delta Lake OSS
+- Delta Lake Databricks
+
+
+## Delta Lake OSS Product tests
+
+Testing against Delta Lake OSS is quite straightforward by simply spinning up
+the corresponding product test environment:
+
+```
+testing/bin/ptl env up --environment 'EnvSinglenodeDeltaLakeOss'
+```
+
+
+## Delta Lake Databricks Product tests
+
+The Databricks Delta Lake implementation provides a richer functionality compared
+to the open source version.
```

Comment:
"At the time of this writing, Databricks Delta Lake and OSS Delta Lake differ in functionality provided."